aequilibrae/distribution/cython/ipf_core.pyx

```diff
@@ -32,44 +33,34 @@ def ipf_core(seed_matrix, target_productions, target_attractions, max_iterations
     cdef double [:] prod_tgt = target_productions
     cdef double [:] prod_factor = factor_prod
 
-    cdef double [:] attr_tot = mat_attr_tot
+    cdef double [::1] attr_tot = np.ascontiguousarray(mat_attr_tot)
```

Comment:
hey @Jake-Moss  - what is the significance/meaning of the `::1` here?

Reply:
That is to declare the memory view as contiguous so we can safely memset it later. Ideally this should be done whenever a memory and C functions interact but it's fine 99% of the time in our usage because we aren't passing in strided numpy arrays

https://cython.readthedocs.io/en/latest/src/userguide/memoryviews.html#pass-data-from-a-c-function-via-pointer
https://cython.readthedocs.io/en/latest/src/userguide/numpy_tutorial.html#declaring-the-numpy-arrays-as-contiguous